consensus/istanbul/validator/weighted.go

```diff
@@ -664,6 +664,30 @@ func (valSet *weightedCouncil) Refresh(hash common.Hash, blockNum uint64, config
 	return nil
 }
 
+// PromoteGoverningNode promotes governing node from demoted validators to staked validators although the node does
+// not have the minimum staking amount.
+func (valSet *weightedCouncil) PromoteGoverningNode(address common.Address) {
+	valSet.validatorMu.Lock()
+	defer valSet.validatorMu.Unlock()
+	var (
+		governingNode *weightedValidator
+		isDemoted     = false
+	)
+	for i, v := range valSet.demotedValidators {
+		if v.Address() == address {
+			isDemoted = true
+			governingNode = v.(*weightedValidator)
+			valSet.demotedValidators = append(valSet.demotedValidators[:i], valSet.demotedValidators[i+1:]...)
+			break
+		}
+	}
+
+	if isDemoted {
```

Comment:
What happened when isDemoted false?

Reply:
Yes, if governing node is not demoted, then nothing happens. The logic is simplified according to https://github.com/klaytn/klaytn/pull/963#discussion_r634174684. Please take another look.